docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -11,3 +11,106 @@ Information that is published here has been reviewed for technical accuracy
 but the format and content is still evolving.
 We hope you will contribute your experiences
 and questions that you have.
+
+The SLIs and SLOs used for Keptn v1 quality gates can be ported to
+KLT Keptn Metrics and Keptn Evaluations.
+
+Note that KLT includes additional observability features
+that are not included in Keptn v1 by default:
+
+* [DORA metrics](../../implementing/dora)
+* [OpenTelemetry observability](../../implementing/otel.md)
+
+Keptn v1 Quality Gates can be migrated into KLT metrics
+and KLT evaluations.
+Notice the paradigm differences:
+
+* Data providers are installed and configured into your Kubernetes cluster
+  using Helm charts and standard practices.
+* You must populate a
+  [KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md) resource
+  for each instance of each data source.
+  This resource specifies the URL and namespace for the data source
+  and gives it a unique `name` that can be referenced by other resources.
+  This is the only KLT specific configuration that is required.
+* Each query that is specified in your Keptn v1
+  [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/) file
+  is converted into its own
+  [KeptnMetric](../../yaml-crd-ref/metric.md) resource.
```

Comment:
With the introduction of analysis the SLO/SLI won't exactly be a keptn metric. Do we want to change this later or do we wait for it to be ready? @thisthat

Reply:
I would immediately change to avoid wrong information. Maybe replace the text with a `TODO` since we don't have all the details yet

Reply:
Okay, I did a bit of rewriting.  This information is explicitly about implementing an SLO that can be implemented as a simple evaluation.  I don't know if anyone wants to do this but they can ;-)  I made that even more explicit and added a note at the beginning that the full SLO functionality is under development and included a link.

While I was at it, I moved the info about implementing a KeptnMetricsProvider to the main how-to page for KeptnMetrics -- what I had written applies to everyone, not just people who are migrating.  And I deleted the other sub-sections since they don't have content and developing that content is not a high priority right now.  So check it out.